### PR TITLE
Added summer vacation notice

### PR DESCRIPTION
**Pull request by benjdlambert**
image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 ![headline](docs/assets/headline.png)
 
+> 🏖 From W/C 11/7 due to Summer Vacations of some of the maintainers, expect the project to move a little slower than normal, and support to be limited. Normal service will resume on W/C 8/8! 🏝
```
Comment (freben) [inline README.md:3]: Honestly I can't parse "W/C 11/7" for several reasons :) Might want to type that out including month name, in both places.

**Review by jhaals — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (jhaals) [reply README.md:3]: I must confess I had to google that ✋

**Review by freben — APPROVED** [review]
<3

**Comment by kuangp** [comment]
Have a good break! 🏖️

**Pull request merged**